### PR TITLE
Ep/min 151 benchmark tests protocol and liquidation pools

### PR DESCRIPTION
**Pull request by Apollo50**
**Description:**

Add benchmark tests for pallets: minterest_protocol, liquidation_pools.
Generate weights files for pallet above.
Add weight annotation for all dispatchable fns in pallets above.

### CODE REVIEW

File: runtime/src/benchmarking/minterest_protocol.rs [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+	use frame_support::assert_ok;
+	use frame_support::traits::GenesisBuild;
+	use liquidity_pools::Pool;
+	use minterest_model::MinterestModelData;
+	use minterest_primitives::Balance;
+	use risk_manager::RiskManagerData;
+	use sp_runtime::traits::Zero;
+
+	// This GenesisConfig is a copy of testnet_genesis.
+	fn new_test_ext() -> sp_io::TestExternalities {
```
Comment (lut1k) [inline runtime/src/benchmarking/minterest_protocol.rs:206]: We use the same genesis block settings in the tests for the protocol and risk-manager.
You can import genesis block settings from the manager to the protocol:
`use crate::benchmarking::risk_manager::tests::new_test_ext;`

Comment (Apollo50) [reply runtime/src/benchmarking/minterest_protocol.rs]: I decided to move genesis block settings to util.rs.
I'm not sure if we need to use the integration test settings.
It is better to use the same genesis config settings as for benchmarks.

**Review by lut1k — APPROVED** [review]

**Comment by kurotychb** [comment]
Should we add a new CI process(job) to check is latest benchmarks were calculated? 
Something like:
1) Make benchmarks tests
2) Compare results with files in benchmarking
3) If results different => reject

**Review by kurotychb — APPROVED** [review]
Should we add a new CI process(job) to check is latest benchmarks were calculated?
Something like:

    Make benchmarks tests
    Compare results with files in benchmarking
    If results different => reject

**Review by dev-vadim — APPROVED** [review]

**Pull request merged**